src/llamacrawl/cli/commands/sessions.py

```
@@ -85,7 +85,10 @@ def _watch_progress(state: CLIState, service: SessionsService) -> None:
             except Exception as e:
                 state.console.print(f"[red]Error:[/red] {e}")
                 state.console.print("[yellow]Retrying in 2 seconds...[/yellow]")
-                logger.error("Failed to fetch sessions progress while watching", extra={"error": str(e)})
+                logger.error(
+                    "Failed to fetch sessions progress while watching",
+                    extra={"error": str(e)},
+                )
```

Ruff (0.14.0)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)